src/components/Post/Post.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Post.scss';
+
+import { CommentList } from '../CommentList';
+import { User } from '../User';
+
+export const Post = ({ title, body, user, comments }) => (
+  <>
+    <h3 className="post__title">{title}</h3>
+    <p className="post__body">{body}</p>
+    <CommentList comments={comments} />
```

Comment:
```suggestion
    <CommentList {...comments} />
```

Reply:
inside comment component is used map method on comments array. I need a variable. Or better to use distr ?

Reply:
oops you are right, keep it as it is)